xbmc/utils/ActorProtocol.h

```diff
@@ -54,10 +55,18 @@ class Message
   Message() {isSync = false; data = NULL; event = NULL; replyMessage = NULL;};
 };
 
+class MessageHandle : public ::KODI::UTILS::CScopeGuard<Message*, nullptr, void(Message*)>
```

Comment:
?? Don't really get what you mean here and how it's related to the line you highlighted